x-pack/plugins/apm/common/rules/apm_rule_types.ts

```diff
@@ -294,6 +298,30 @@ export type AnomalyAlertSeverityType = ValuesType<
   typeof ANOMALY_ALERT_SEVERITY_TYPES
 >['type'];
 
+export const ANOMALY_DETECTOR_SELECTOR_OPTIONS = [
+  {
+    type: ApmMlDetectorType.txLatency,
+    label: i18n.translate('xpack.apm.alerts.anomalyDetector.latencyLabel', {
+      defaultMessage: 'latency',
+    }),
+  },
+  {
+    type: ApmMlDetectorType.txThroughput,
+    label: i18n.translate('xpack.apm.alerts.anomalyDetector.throughputLabel', {
+      defaultMessage: 'throughput',
+    }),
+  },
+  {
+    type: ApmMlDetectorType.txFailureRate,
+    label: i18n.translate(
+      'xpack.apm.alerts.anomalyDetector.failedTransactionRateLabel',
+      {
+        defaultMessage: 'failed transaction rate',
+      }
+    ),
+  },
+] as const;
```

Comment:
```suggestion
export const ANOMALY_DETECTOR_SELECTOR_OPTIONS = [
  ApmMlDetectorType.txLatency,
  ApmMlDetectorType.txThroughput,
  ApmMlDetectorType.txFailureRate,
].map((type) => ({ type, label: getApmMlDetectorLabel(type) }));
```

Reply:
Does `ANOMALY_DETECTOR_SELECTOR_OPTIONS` need to be in `common`? It's only used on the client

Reply:
Actually I was thinking to move the `getApmMlDetectorLabel()` to common, as we will need to use the labels also for the `formatAnomalyReason()`

Reply:
That makes sense if we need it on the server side as well 👍